packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -147,7 +147,10 @@ const FORCE_ERROR_RESET = /*          */ 0b100;
 type FiberInstance = {
   kind: 0,
   id: number,
-  parent: null | DevToolsInstance, // virtual parent
+  parent: null | DevToolsInstance, // filtered parent, including virtual
+  firstChild: null | DevToolsInstance, // filtered first child, including virtual
+  previousSibling: null | DevToolsInstance, // filtered next sibling, including virtual
```

Comment:
Previous sibling is really only used when we move something out of the remaining children set. If we did implement the reorder algorithm like we do in Fiber by using a temporary map when things are reordered then we wouldn't need this extra field so we can get rid of it later.